WcaOnRails/spec/i18n_locale_files_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'i18n-spec'
+
+Dir.glob(Rails.root.join('config', 'locales', '*.yml')).each do |locale_file|
+  RSpec.describe locale_file.to_s do
+    it { is_expected.to have_valid_pluralization_keys }
```

Comment:
> Upon checking Travis, the results of these tests show up in disparate places. See here and here for instance. Would it be possible to consolidate them by adding a top level describe or context to this whole test?
Oops that's ugly! I didn't notice it because I ran only this check locally, but I'll change this!